### PR TITLE
Format blocks and labeled statements.

### PR DESCRIPTION
**Pull request by munificent**
This doesn't support blank lines between statements in blocks. That will happen when comments are supported, which this is leading up to.

### CODE REVIEW

**Comment by natebosch** [comment]
I think the merge base and the commits on this branch got out of alignment - do you want me to try to find a commit range to review a diff in this PR, or should we wait until other commits settle and we can merge in the right commits to have a clean diff here?

**Comment by munificent** [comment]
I'll resync the branch. The `format-binary` is the next one in the chain to review and this one comes after that, followed by `format-comments`. (The PR number order should tell you what order to review them.)

**Comment by munificent** [comment]
OK, this should be in a reviewable state now.

**Review by natebosch — APPROVED** [review]

**Pull request merged**
